### PR TITLE
Prepare dist build

### PR DESCRIPTION
**Pull request by MichalZalecki**
Hi, 

really cool concept! When I wanted to test it I've noticed after installing package it's not possible to require module as `main` entry in `package.json` was invalid and there wasn't real build to make it available among different build tools.

Cheers!

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["node6"]
+  "presets": ["es2015"]
```
Comment (FredyC) [inline .babelrc]: I would recommend using this preset instead: https://github.com/gabmontes/babel-preset-latest-minimal

Comment (MichalZalecki) [inline .babelrc]: In what way it's going to make it any better?

**Comment by ericelliott** [comment]
Hi,

Please feel free to PR.

Here's the issue:

1. `package.json` needs to point to `source/dsm.js`
2. This is an ES6 module, meaning that you can't just import it in Node and start using it. You have to transpile or use `babel-core/register` prior to attempting to import it. Since this is a general-purpose library, I'd be happy to switch to the commonjs module format, which AFAIK, is supported natively by Node, browserify, webpack, and JSPM, which I think covers all of the most common package managers in use today.

All we need to do to fix #2 is swap out `import`/`export` with `require`/`module.exports`.

File: .babelrc (hunk 1 above)
Comment (ericelliott) [inline .babelrc]: This is not necessary. Babel is used only for tests, and end users can use whatever babel config they want (provided it's compatible) with the features we use and the targets they want to support.

**Comment by zouhir** [comment]
I can work on those.

**Comment by ericelliott** [comment]
Closing -- replacing with [Fix imports #4](https://github.com/ericelliott/redux-dsm/issues/4).

**Pull request closed without merging**